src/io/flutter/logging/FlutterLogFilterPanel.java

```diff
@@ -80,15 +82,15 @@ public int hashCode() {
   private SearchTextField textExpression;
   private JComboBox<FlutterLog.Level> logLevelComboBox;
 
-  public FlutterLogFilterPanel(
-    @NotNull OnFilterListener onFilterListener
-  ) {
+  public FlutterLogFilterPanel(@NotNull OnFilterListener onFilterListener) {
     this.onFilterListener = onFilterListener;
+
     matchCaseCheckBox.addItemListener(e -> onFilterListener.onFilter(getCurrentFilterParam()));
     regexCheckBox.addItemListener(e -> onFilterListener.onFilter(getCurrentFilterParam()));
     final List<FlutterLog.Level> logLevels = Arrays.stream(FlutterLog.Level.values())
       .collect(Collectors.toList());
     logLevelComboBox.setModel(new CollectionComboBoxModel<>(logLevels));
+    logLevelComboBox.setSelectedItem(FlutterLog.Level.CONFIG);
```

Comment:
I just confused.
In https://github.com/flutter/flutter-intellij/pull/2441/files#diff-c8d60b1f6627b5f4700899a7d7f97b88R36
we defined:
```java
public static int GC_EVENT_LEVEL = FlutterLog.Level.FINER.value;
```
But here you make default selected item is `Level.CONFIG`

=> what's the different ?

Reply:
GC events are printed at a slightly more verbose setting than the default. While using the logging view, I found GC events noisy and not adding a ton of value.

We should either filter them out by severity, or add the ability to show and hide categories (gc event, network events, ...).

Reply:
>  or add the ability to show and hide categories 

👍 I'd like to see this happen.  https://github.com/flutter/flutter-intellij/issues/2445 to track.